packages/common/src/types/command/ActionDescriptor.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  PartialDestinationDescriptor,
+  PartialTargetDescriptor,
+} from "./PartialTargetDescriptor.types";
+
+const simpleActionNames = [
+  "callAsFunction",
+  "clearAndSetSelection",
+  "copyToClipboard",
+  "cutToClipboard",
+  "deselect",
+  "editNew",
+  "editNewLineAfter",
+  "editNewLineBefore",
+  "executeCommand",
+  "extractVariable",
+  "findInWorkspace",
+  "foldRegion",
+  "followLink",
+  "generateSnippet",
+  "getText",
+  "highlight",
+  "indentLine",
+  "insertCopyAfter",
+  "insertCopyBefore",
+  "insertEmptyLineAfter",
+  "insertEmptyLineBefore",
+  "insertEmptyLinesAround",
+  "insertSnippet",
```

Comment:
Ideally this would accept a destination, but maybe it's simpler to start with a target for now? Or maybe you just haven't implemented that part yet 😅